spyder/plugins/editor.py

```diff
@@ -1929,7 +1929,7 @@ def print_file(self):
                           header_font=self.get_plugin_font('printer_header'))
         printDialog = QPrintDialog(printer, editor)
         if editor.has_selected_text():
-            printDialog.addEnabledOption(QAbstractPrintDialog.PrintSelection)
+            printDialog.setOption(QAbstractPrintDialog.PrintSelection, True)

```

Comment:
@dalthviz is this a change on Qt5 ? or ?, should we had some try/except based on version number?